frontend/src/document/document.js

```diff
@@ -76,6 +82,11 @@ module.exports = app => app.component('document', {
       this.changes = {};
       this.editting = false;
     },
+    async confirmSave() {
+      this.shouldShowConfirmModal = true;
+      await sleep(50); // errors without it
```

Comment:
This is a bit hacky. The better way would be to create a new `confirm-changes` component that contains the body of the `shouldShowConfirmModal` modal, and call `Prism.highlightElement()` in that component's `mounted` hook